index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
```

Comment:
## OPTIONAL CHANGE: FIX the width of the entire page
- I believe the width of the entire page should be getting a `container` class as opposed to it having `container-fluid` class for it to resemble its clone as shown below. Do you mind making this one final change before we proceed?

|Source Page|Clone Page|
|:--:|:--:|
|![source](https://user-images.githubusercontent.com/22345961/114099888-31221680-98cc-11eb-9a11-8cf3260fee9d.png)|![clone](https://user-images.githubusercontent.com/22345961/114099910-397a5180-98cc-11eb-9f69-c9f9794a28a6.png)|